SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Common/LinuxScripts.cs

```diff
@@ -55,6 +55,13 @@ shift 1
 
         public const string SimulatedPlayerCoordinatorShellScript =
 @"#!/bin/sh
+
+# Some clients are quite large so in order to avoid running out of disk space on the node we attempt to delete the zip
+WORKER_ZIP_DIR=""/tmp/runner_source/""
+if [ -d ""$WORKER_ZIP_DIR"" ]; then
+  rm -rf ""$WORKER_ZIP_DIR""
```

Comment:
```suggestion
  rm -rf ""${WORKER_ZIP_DIR}""
```

Reply:
I don't see a reason to use variable expansion here. Could you justify?